guide/english/certifications/front-end-libraries/react/create-a-controlled-input/index.md

```diff
@@ -31,5 +31,6 @@ It can be a bit hard to digest but to make things further clear try removing the
 ```
 <input value = {this.state.input}/>
 ```
-Now run the tests again are you able to type anything? <br>
-The answer to it will be "NO" since your input box is getting value from the state variable `input` since there is no change in the state `input`(an empty string initially) which will only happen when you trigger the function `handleChange()` which will only happen when you have an event handler like `onChange()` hence the string inside the input box will remain as it is i.e, an empty string.
+
+Now run the tests again. Are you able to type anything? <br>
+The answer to the above question will be "NO". Your input box is getting value from the state variable `input`. Since there is no change in the state of `input` (an empty string initially), nothing will happen. A change will only occur when you trigger the function `handleChange()`.  This function will only be triggered when you have an event handler like `onChange()`. If there is no `handleChange()` method, the string inside the input box will remain as it is i.e, an empty string.
```

Comment:
My only suggestion is to surround the `i.e, an empty string` with `(    )`.
```suggestion
The answer to the above question will be "NO". Your input box is getting value from the state variable `input`. Since there is no change in the state of `input` (an empty string initially), nothing will happen. A change will only occur when you trigger the function `handleChange()`.  This function will only be triggered when you have an event handler like `onChange()`. If there is no `handleChange()` method, the string inside the input box will remain as it is (i.e, an empty string).
```